view-user/src/components/common/Modal/Modal.module.css

```
@@ -0,0 +1,83 @@
+.wrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  contain: content;
+}
+
+.content {
+  position: relative;
+  z-index: 1;
+  box-sizing: border-box;
+  object-fit: cover;
+  padding: 40px;
+  overflow: hidden;
+  background-color: #c67979;
+  box-shadow: 0 0 0 3.56rem #d9d9d9;
+  border-radius: 5px;
+  animation: zoomIn 0.8s cubic-bezier(0.25, 1, 0.5, 1) 1 forwards;
+  will-change: transform, opacity;
+}
+
```

```suggestion

@media screen and (max-width: 460px) {
  .content {
    box-shadow: 0 0 0 1.5rem #d9d9d9;
  }
}
```
スマホの画面でbox-shadowがついちゃうのが気になったので修正しました